analysis/export/CSVExporter/README.md

```diff
@@ -17,8 +25,14 @@ _Conventions for csv output:_
 
 ### Example
 
-> ccsh csvexport visual.cc.json
+With output in CLI:
+
+`ccsh csvexport visual.cc.json`
+
+The following generates result.csv as output, while truncating path lists after length 4:
+
+`ccsh csvexport visual.cc.json --depth-of-hierarchy=4 -o result.csv`
```

Comment:
Here we could add one or two example csv lines from result.csv to demonstrate truncated pathes.

Reply:
Resolved by commit [9eb6073](https://github.com/MaibornWolff/codecharta/pull/2927/commits/9eb6073274035f15524abecd9b8d4f5ec39867cd)